blocks/leaphy/leaphy_common.js

```diff
@@ -26,50 +26,60 @@ goog.require('Blockly.Mutator');
 
 Blockly.Blocks['leaphy_start'] = {
```

Comment:
Curious why you did not define this block fully as json. If possible, move it to a fully json block definition inside `Blockly.defineBlocksWithJsonArray(...)`

Reply:
Thought it wasn't possible, but I got it working with extensions.